docs/user_guide/plotting.md

```diff
@@ -1,11 +1,341 @@
+## Creating your first plot
 
-TODO
+Assuming you used the analysis template to setup your analysis, you can create a first plot by running
+```shell
+law run cf.PlotVariables1D --version v1 \
+    --calibrators example --selector example --producer example \
+    --processes data,tt,st --variables n_jet --categories incl,2j
+```
+This will run the full analysis chain for the given processes (data, tt, st) and should create
+plots looking like this:
 
-## Variables creation
 
-see the Variables creation section in the Config Objects section of the documentation
+::::{grid} 1 1 2 2
+:::{figure} ../plots/cf.PlotVariables1D_tpl_config_analy__1__12dfac316a__plot__proc_3_7727a49dc2__cat_incl__var_n_jet.pdf
+:width: 100%
+:::
+
+:::{figure} ../plots/cf.PlotVariables1D_tpl_config_analy__1__12dfac316a__plot__proc_3_7727a49dc2__cat_2j__var_n_jet.pdf
+:width: 100%
+:::
+::::
+
+
+:::{dropdown} Where do I find that plot?
+You can add ```--print-output 0``` to every task call, which will print the full filename of all
+outputs of the requested task. Alternatively, you can add ```--fetch-output 0,a``` to directly
+copy all outputs of this task into the directory you are currently in.
+Finally, there is the ```--view-cmd``` parameter you can add to directly display the plot during
+the runtime of the task, e.g. via ```--view-cmd evince-previewer```.
+:::
+
+
+The PlotVariables1D task is located at the bottom of our
+[task graph](https://github.com/columnflow/columnflow/wiki#default-task-graph), which means that
+all tasks leading to PlotVariables1D will be run for all datasets corresponding to the
+```--processes``` we requested using the Calibrators, Selector, and Producers
+(often referred to as CSPs) as requested. In the following examples, we will skip the
+```--calibrators```, ```selector``` and ```producers``` parameters, which means that the defaults
+defined in the config will be used automatically. Examples on how to
+implement your own CSPs can be found in the [calibrators](building_blocks/calibrators),
+[selectors](building_blocks/selectors), and [producers](building_blocks/producers) sections of the
+user guide. The ```--variables``` parameter defines, for which variables we want to create histograms
+and plots. Variables are order objects that need to be defined in the config as shown in the
+[config objects](building_blocks/config_objects) section and the column corresponding to the expression
+statement needs to be stored either after the {py:class}`~columnflow.tasks.reduction.ReduceEvents`
+For each of the category given with the ```--categories``` parameter, one plot will be produced.
+A detailed guide on how to implement categories in Columnflow is given in the
+[categories](building_blocks/categories) section.
+
+To define, which processes and datasets to consider when plotting, you can use the ```--processes```
+and ```--datasets``` parameter. When only processes are given, all datasets corresponding to the
```

Comment:
I'm not sure if a agree, because we use the words datasets and processes quite often.
I'd only highlight processes/datasets, if I'm actually talking about a `Process` object or about the `--processes` parameter